CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ Changes since v2.16
 - A new form field type "table" is now available. A table has a predefined set of columns, and applicants can fill in as many rows as they wish. **You can not roll back to an earlier release once your database contains applications with filled-in table fields. You will need to fix the database manually.** (#2551)
 
 ### Changes
+- Form titles are now deprecated. Prefer the new `internal-name` or name and `external-title` or title fields instead. The external title is shown to applicants in the heading of the form and internal name used throughout the administration. The API supports the old style for now. (#2066)
```

Comment:
"Form titles are now deprecated" sets this off on the wrong foot I think. How about something like

- Show localized form titles to applicants. The field previously called form `title` is now `internal-name`, and there is a new `external-title` field that is a localized public title for the form. ...

Reply:
Yeah I can do that. I agree it's a rather strong wording of an actual improvement.